OpenDreamRuntime/Objects/Types/DreamObjectAtom.cs

```diff
@@ -38,6 +31,12 @@ protected override void HandleDeletion(bool possiblyThreaded) {
 
         base.HandleDeletion(possiblyThreaded);
     }
+    public string GetDesc() {
```

Comment:
Some spacing between methods, but this should also be renamed to `GetRTEntityDesc()` to make it clearer that it's only used for RT's entities.

```suggestion
    }
    
    public string GetDesc() {
```